scripts/ci/docker-compose.yml

```diff
@@ -0,0 +1,84 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "2.2"
+services:
+  mysql:
+    image: mysql:5.6
+    restart: always
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=true
+      - MYSQL_ROOT_HOST=%
+    volumes:
+      - ./mysql/conf.d:/etc/mysql/conf.d
+
+  postgres:
+    image: postgres:9.6
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=airflow
+      - POSTGRES_DB=airflow
+
+  cassandra:
+    image: cassandra:3.0
+    restart: always
+
+  rabbitmq:
+    image: rabbitmq:3.7
+    restart: always
+
+  openldap:
+    image: osixia/openldap:1.2.0
+    restart: always
+    command: --copy-service
+    environment:
+      - LDAP_DOMAIN=example.com
+      - LDAP_ADMIN_PASSWORD=insecure
+      - LDAP_CONFIG_PASSWORD=insecure
+    volumes:
+      - ./openldap/ldif:/container/service/slapd/assets/config/bootstrap/ldif/custom
+
+  krb5-kdc-server:
+    build: ./krb5
+    image: krb5-kdc-server
+    hostname: krb5-kdc-server
+    domainname: example.com
+
+  airflow-testing:
+    image: gerardo/airflow-testing:latest
```

Comment:
Please change to non-personal repo

Reply:
@dimberman I'll change it to `apache/airflow-testing`, but we need an image living there first. 

The repo where the current Dockerfile lives is https://github.com/gerardo/incubator-airflow-ci. 
And here's the current TravisCI build: https://travis-ci.org/gerardo/incubator-airflow-ci

Reply:
BTW, this is the same process used by the CouchDB project: https://travis-ci.org/apache/couchdb-ci

Reply:
Requested under https://issues.apache.org/jira/browse/INFRA-16905

Reply:
Hi Gerardo, it took a while. But the docker image is being build here: https://hub.docker.com/r/airflowci/incubator-airflow-ci/builds/

Based on this repository: https://github.com/apache/incubator-airflow-ci

Reply:
@gerardo Could you update the PR?